### ISSUE TITLE
OSD date/time is corrupt

### ISSUE THREAD

**Issue by wayne47git** [openipcamera/openipc-firmware#50]
VLC screenshot:

![image](https://user-images.githubusercontent.com/25324850/39766975-9b8e4d2a-52b3-11e8-9f97-87fa487e23a0.png)

[root@openipc:~]# date
Tue May  8 11:32:53 EDT 2018

**Comment by PH89** [comment]
What date format string you used?

**Comment by wayne47git** [comment]
Good catch - typo, I had %s, not %d. But it still should repaint correctly?
%m/%s/%y %H:%M:%S
